### DISCUSSION TITLE
QuarkusDev fails without a clear error

### DISCUSSION THREAD

**Discussion by LarsSven** [quarkusio/quarkus#42295] [Q&A]
So I am working on the following project using Quarkus 3.12, Kotlin and Gradle: https://gitlab.com/rug-digitallab/products/themis/runtimes/container-runtime

When I try to run `:container-runtime:quarkusDev`, I get the following error: 
```
[Quarkus build analytics] Unexpected error reading class java.lang.NoSuchFieldError from path: C:\Users\cactu\.redhat\io.quarkus.analytics.remoteconfig. Got message: READ_DATE_TIMESTAMPS_AS_NANOSECONDS. Attempting to continue...
```

This is the only information I get. When I actually run the project without Quarkus Dev (like building it in a container and running it), it does work. But when I try to run it as Quarkus Dev, it fails with this error.

Does anyone know why this happens and how I can fix it?

[a comment by an automated account is not shown]

**Reply by geoand** [reply]
cc @brunobat

**Comment by gsmet** [comment]
I suspect a version mismatch somewhere but it's hard to figure it out from your project structure. Check that the Quarkus Gradle plugin and the Quarkus dependencies are exactly the same version.

**Reply by LarsSven** [reply]
It turns out it was a dependency that uses an ancient version of Jackson (2.10) that is loaded into the buildSrc of the project that was causing the issue

**Reply by LarsSven** [reply]
Adding this to the buildSrc dependencies resolved the problem for me (though it is very much a workaround of course)

```
implementation("com.fasterxml.jackson.core:jackson-databind:2.17.2")
implementation("com.fasterxml.jackson.core:jackson-core:2.17.2")
implementation("com.fasterxml.jackson.core:jackson-annotations:2.17.2")
```

**Reply by gsmet** [reply]
Don't you enforce the Quarkus BOM?

**Comment by mageddo** [comment]
In my case quarkus plugin depends on jackson and it was conflicting with org.flywaydb.flyway plugin, upgraded flyway plugin and the problem was gone
